.gitignore

```diff
@@ -1,4 +1,6 @@
-/shopify
+.byebug_history
+schema/attribute_data.cue
+
 /_site
 /docs/_data
-.byebug_history
+/shopify
```

Comment:
I don't see this folder in `main` or the `gh-pages` branch. Still needed?

Reply:
Ah this was here before, but looks to be from the early days. Will 🔥